code/controllers/subsystem/atoms.dm

```diff
@@ -28,17 +28,18 @@ SUBSYSTEM_DEF(atoms)
 	initialized = INITIALIZATION_INNEW_MAPLOAD
 
 	var/count
-	var/list/mapload_arg = list(TRUE)
+	var/static/list/mapload_arg = list(TRUE)
```

Comment:
Reasoning? The memory loss lasts a lot longer than the performance gain on startup

Reply:
isn't there a contribution guideline about statics/shouldn't this just live on GLOB

Reply:
Well its a var that is only used in this one case to tell InitAtoms that its map loading.  Its always sent list(TRUE) all the time always.  Thought it be better as a static than a new list evey run

Reply:
This is still a thing